src/ImageSharp/Processing/Delegates/PixelShader.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Numerics;
+
+namespace SixLabors.ImageSharp.Processing.Delegates
```

Comment:
Done in https://github.com/SixLabors/ImageSharp/pull/1065/commits/62272a32831f4d67df876a74e6351e076863a4aa. Though maybe it could be a good idea to separate delegates, enums and other types under `SixLabors.ImageSharp.Processing` into different subfolders in the future, for clarity? 🤔
That path is pretty crowded right now, with heterogeneous types.

Reply:
>Though maybe it could be a good idea to separate delegates, enums and other types under SixLabors.ImageSharp.Processing into different subfolders in the future, for clarity?

We actually used to have them in separate namespaces in the same manner as the individual processors but moved them because people were struggling with discover-ability. I could very easily be convinced to return to that pattern though. 

@SixLabors/core Thoughts?

Reply:
There's also the middle ground (although maybe less "clean") to separate the files in subfolders while still keeping them all in the same namespace, for user discoverability.
I'll wait for feedback from you guys 😄

Reply:
Either way a separate PR so don't worry about it.

Regarding tests. Creating a reference image would be best as that lets us visually confirm the output also.